src/components/DuffelCardForm/lib/postMessageToStoreCardForTemporaryUse.ts

```diff
@@ -0,0 +1,22 @@
+export function postMessageToStoreCardForTemporaryUse(
```

Comment:
```suggestion
export function postMessageToCreateCardForTemporaryUse(
```
If we want to be totally consistent with the wording...

Reply:
Great shout, I think we do!